index.html

```diff
@@ -0,0 +1,352 @@
+        </h1>
+      </div>
+
+      <div class="container mainPara text-black my-3">
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit ad
+          laudantium voluptatum, modi natus perspiciatis. Lorem ipsum dolor sit
+          amet consectetur adipisicing elit.
+        </p>
+      </div>
+      <div class="container mt-3">
+        <h1>
+          2022.10.20(THU) ~ 21(FRI)
+        </h1>
+        <h3 style="padding-bottom: 10px; margin: 0;">
+          @ Lorem ipsum dolor sit amet consectetur adipisicing elit.
+        </h3>
+      </div>
+    </div>
+
+    <!-- // Program Section // -->
+
+    <div id="main2" class="container-fluid">
+      <div class="container text-white p-3 mb-5">
+        <h3 id="prgrmHearder">
+          Main Program
+        </h3>
+        <hr id="prgrmHr" />
+      </div>
+      <div class="row">
+        <div id="prgrmCol1" class="col prgrmCol">
+          <div class="card p-3 prgrmCard">
+            <div class="card-img">
+              <i
+                class="material-icons"
+                style="font-size: 50px; color: rgb(226, 215, 215);"
+              >
+                perm_camera_mic
+              </i>
+            </div>
+            <div>
+              <h4 class="card-title prgrmTitle">monolog</h4>
+            </div>
+            <div class="card-body">
+              <p class="card-text" style="font-size: small;">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Cum,
+                ipsa?
+              </p>
+            </div>
+          </div>
+        </div>
+
+        <div class="col prgrmCol">
+          <div class="card p-3 prgrmCard">
+            <div class="card-img">
+              <i
+                class="material-icons"
+                style="font-size: 50px; color: rgb(226, 215, 215);"
+              >
+                ssid_chart
+              </i>
+            </div>
+            <div>
+              <h4 class="card-title prgrmTitle">Exhibition</h4>
+            </div>
+            <div class="card-body">
+              <p class="card-text" style="font-size: small;">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Cum,
+                ipsa?
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="col prgrmCol">
+          <div class="card p-3 prgrmCard">
+            <div class="card-img">
+              <i
+                class="material-icons"
+                style="font-size: 50px; color: rgb(226, 215, 215);"
+              >
+                dns
+              </i>
+            </div>
+            <div>
+              <h4 class="card-title prgrmTitle">Platforum</h4>
+            </div>
+            <div class="card-body">
+              <p class="card-text" style="font-size: small;">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Cum,
+                ipsa?
+              </p>
+            </div>
+          </div>
```

Comment:
- [ ] Kindly align these articles arccording to the original design

Your design
![image](https://user-images.githubusercontent.com/81932478/205464921-edda4284-068e-4ff4-a5d8-cf7226722996.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/205464932-f081e254-4e4f-4bc3-ac34-fc7479899e22.png)

Reply:
They are working fine on both given sizes. But not for other sizes.

Reply:
- [ ] I think here the same issue so please align it right, and I think since your design should be responsive so it should be aligned right in all the sizes not in specific sizes so kindly fix the design here to match the original design please as requested by the previous code reviewer ✨🔥💯


- [ OPTIONAL ] I think you didn't add the animation as show in the original design so when the user press on each card it has white border as shown so I recommend to add it here so make your design has  human touch 💯💯💯


- Your design
![image](https://user-images.githubusercontent.com/58553711/205466263-49ac7e1f-5173-4afe-9807-d29bf618723b.png)

- Original design
 ![image](https://user-images.githubusercontent.com/81932478/205464932-f081e254-4e4f-4bc3-ac34-fc7479899e22.png)